src/neo/Trie/MPT/MPTNode/FullNode.cs

```diff
@@ -0,0 +1,58 @@
+using System.IO;
+using Neo.Cryptography;
+using Neo.IO;
+
+namespace Neo.Trie.MPT
+{
+    public class FullNode : MPTNode
+    {
+        public MPTNode[] Children = new MPTNode[17];
```

Comment:
plz, use a const variable to replace the `17`

Reply:
👌